Exiled.CustomItems/API/Features/CustomGrenade.cs

```diff
@@ -156,9 +156,6 @@ private void OnInternalThrowing(ThrowingGrenadeEventArgs ev)
 
             OnThrowing(ev);
 
-            if (!ev.IsAllowed)
```

Comment:
Why remove this? It could be useful for items that want to handle the throwing on their own, or want to just not spawn a grenade that's thrown.

Reply:
Because it would never allow to throw grenades, since it's set to false 3 lines above.

Reply:
Oh good point. Should we add a way to have the built in OnThrowing() event tell the CustomGrenade OnThrowing event not to continue, then?

For example with the Custom Item "Rock", I don't want the CustomGrenade OnInternalThrowing() to continue after it calls the virtual OnThrowing method. The only way to do that currently is override SubscribeEvents to not have it subscribe to InternalThrowing.

Reply:
Yeah, we could.